src/getDataset.js

```diff
@@ -4,6 +4,7 @@ const helpers = require("./getDatasetHelpers");
 const {NoDatasetPathError} = require("./exceptions");
 const auspice = require("auspice");
 const request = require('request');
+const assert = require('assert');
```

Comment:
Ah, nice, I didn't know about this core NodeJS library. When I looked it up (thinking it was a third-party library), I noticed that the official docs suggest using strict equality by default. This makes sense, as we also prefer `===` vs. `==`. Looks like it should be a small change to use strict equality, by doing:

    const assert = require('assert').strict;

https://nodejs.org/docs/latest-v13.x/api/assert.html#assert_strict_assertion_mode

Reply:
Thanks, Tom. I added that into my commits.